ceph-pr-commits/config/definitions/ceph-pr-commits.yml

```diff
@@ -0,0 +1,68 @@
+- job:
+    name: ceph-pr-commits
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph: Pull Request commits'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          admin-list:
```

Comment:
This could be a much more liberal list - I think the above Python code should be safe to run on anything.